Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/weatherInput/index.tsx

```diff
@@ -1,4 +1,9 @@
-const WeatherInput = () => {
+const WeatherInput = (props: {
+  setWeather: React.Dispatch<React.SetStateAction<number>>;
+}) => {
+  const handleWeatherChange = (newValue: number) => {
+    props.setWeather(newValue);
+  };
   return (
     <>
       <img src="/Cloud.png" height="66px" width="66px" alt="Cloud" />
```

Comment:
We should probably bind the input field to our state variable we created in the parent component